src/main/java/internetshop/dao/hibernate/ItemDaoHibernateImpl.java

```diff
@@ -0,0 +1,74 @@
+package internetshop.dao.hibernate;
+
+import internetshop.annotation.Dao;
+import internetshop.dao.ItemDao;
+import internetshop.model.Item;
+import internetshop.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+import java.util.List;
```

Comment:
@SkaterWeather have you run mvn clean package?

Reply:
yeah, all is ok, there is only checkstyle warnings type of "Wrong lexicographical order ..."
- it's in classes that i wrote before configiring import settings
- i didn't want to fix this in pull request, because if i would do that, there was a 15-20 files with small non-sense changes -> and you have to check them all (definetly sounds not good)
- - - - - - - - - 
i was planning to fix them after aproving the request, so i will don't waste your time scrolling and checking all of that.